source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -166,219 +255,227 @@ function New-StigCheckList
 
     #endregion ASSET
 
+    #region STIGS
     $writer.WriteStartElement("STIGS")
-    $writer.WriteStartElement("iSTIG")
-
-    #region STIGS/iSTIG/STIG_INFO
-
-    $writer.WriteStartElement("STIG_INFO")
-
-    $xccdfBenchmarkContent = Get-StigXccdfBenchmarkContent -Path $xccdfPath
-
-    $stigInfoElements = [ordered] @{
-        'version'        = $xccdfBenchmarkContent.version
-        'classification' = 'UNCLASSIFIED'
-        'customname'     = ''
-        'stigid'         = $xccdfBenchmarkContent.id
-        'description'    = $xccdfBenchmarkContent.description
-        'filename'       = Split-Path -Path $xccdfPath -Leaf
-        'releaseinfo'    = $xccdfBenchmarkContent.'plain-text'.InnerText
-        'title'          = $xccdfBenchmarkContent.title
-        'uuid'           = (New-Guid).Guid
-        'notice'         = $xccdfBenchmarkContent.notice.InnerText
-        'source'         = $xccdfBenchmarkContent.reference.source
-    }
 
-    foreach ($StigInfoElement in $stigInfoElements.GetEnumerator())
+    #region STIG_iteration
+    foreach ($XccdfPathItem in $XccdfPath)
```

Comment:
$XccdfPathItem should begin with a lower case 'x'

Reply:
Done.